mkdocs.yml

```diff
@@ -189,7 +188,8 @@ nav:
       - OBOOK Maturity Indicator: reference/obook-maturity-indicator.md
       - ChatGPT:
           - Leveraging ChatGPT for ontology curation: reference/chatgpt-prompts-for-ontology-development.md
-  - Explanations:
+      - Ontology User references: reference/ontology-user.md 
```

Comment:
is there a reason you say "references" rather than "reference" here? is it because you intend to add more references later?